config/helm/chart/default/templates/Common/csi/daemonset.yaml

```diff
@@ -170,6 +170,9 @@ spec:
           - name: MAX_UNMOUNTED_VOLUME_AGE
             value: "{{ .Values.csidriver.maxUnmountedVolumeAge}}"
           {{- end }}
+        envFrom:
```

Comment:
why is this necessary? if `csiDriver` is not enabled, the daemonset does not exist